cimi-resources/src/com/sixsq/slipstream/ssclj/resources/external_object.clj

```diff
@@ -409,9 +383,9 @@
           (fn [resource _] (:objectType resource)))
 
 (defmethod delete-subtype :default
```

Comment:
I think that this implementation does not treat correctly the case where the S3 delete request throws an exception; at the very least an informative message should be added to the thrown exception before propagating this back up the stack. 

I also don't think that the logic is correct when the underlying bucket/object doesn't exist.  In this case, the flag will indicate that the external object can be deleted and then the code will actually try to delete the non-existent bucket/object, which will then throw an exception that causes the delete request to fail.

Also, I wonder whether there's any benefit is having this as a multi-method.  Neither of the existing implementations specialize the behavior and generally I wouldn't see where this is necessary.

Reply:
When bucket does exists, the cimi resource deletion now succeeds. A 403 is issued when the bucket ACL does not allow it.